src/components/navbarLinks/navbarLinksMobile.tsx

```diff
@@ -0,0 +1,29 @@
+import slugify from 'slugify';
+import style from 'src/layouts/general/layout.module.css';
+import { and } from 'src/utils/css';
+const NavbarLinksMobile = (props: {
+  heading: TocItem;
+  tabIndex: number;
+  isOpen: boolean;
+}) => {
```

Comment:
```suggestion
type NavbarLinksMobileProps = {
  heading: TocItem;
  tabIndex: number;
  isOpen: boolean;
};

const NavbarLinksMobile = ({
  heading,
  tabIndex,
  isOpen,
}: NavbarLinksMobileProps) =>
```

Kan gjøre koden litt mer oversiktlig og man slipper å skrive props. ... 

Kan se på denne: https://blog.variant.no/a-better-way-to-type-react-components-9a6460a1d4b7

Reply:
Ser forresten at begge NavbarLinks kunne brukt samme type/interface her